### ISSUE TITLE
Virtual network cam driver work only for skype  

### ISSUE THREAD

**Issue by rajdipsimform** [tedd/Tedd.VirtualNetworkCam#5]
Hello there 
I have installed driver successfully, camera name is showing all over the place but only work with skype
when i start skype its showing logs
![image](https://user-images.githubusercontent.com/89386681/148931143-ab59110c-4390-438a-a913-e8f394894ace.png)
which its says its started listing, but when i stop skype and other app try to access camera its can't get image or feed.
virtual camera is working for only skype only. any idea how i can make it work for other application

My main goal is show camera frame to virtual camera in c#(WPF)

**Comment by BakingBrains** [comment]
@rajdipsimform were you able to resolve this?

**Comment by rajdipsimform** [comment]
@BakingBrains no, still i am facing issue

**Comment by BakingBrains** [comment]
@rajdipsimform I think zoom only accepts "Zoom verified drivers".
check this link : [https://community.zoom.com/t5/Rooms-and-Workspaces/New-third-party-USB-HID-device/m-p/9318#M188](https://community.zoom.com/t5/Rooms-and-Workspaces/New-third-party-USB-HID-device/m-p/9318#M188)

This might be the issue. I am not sure

**Comment by rajdipsimform** [comment]
@BakingBrains i am looking for generic driver for all applications, virtual driver need to be available for all the application as camera name, just like OBS studio doing. do you think we can achieve this by C#?

**Comment by BakingBrains** [comment]
@rajdipsimform Yes that is possible. The thing is we need to find a way to send the integrated camera frames to the developed filter driver.

**Comment by rajdipsimform** [comment]
@BakingBrains  do you have any idea or code snip for the same 

> @rajdipsimform Yes that is possible. The thing is we need to find a way to send the integrated camera frames to the developed filter driver.
